examples/lock-app/efr32/include/LockManager.h

```diff
@@ -31,15 +31,26 @@
 
 using namespace ::chip;
 
-// Currently up to 10 users are support on the EFR32 platform
+// up to 10 users are supported on the EFR32 platform
 #define DOOR_LOCK_MAX_USERS 10
+// max credential size supported is 8
```

Comment:
I somehow find that these comments re-read the code:

```
#define DOOR_LOCK_MAX_CREDENTIAL_SIZE 8
```

clearly reads "the maximum credential size for the door lock is 8". Then the comment says `max credential size supported is 8` ... that seems like a waste of vertical space when reading code.

Please update comments to include only things that help reading the code. Just re-stating things especially numbers means we are just forcing any future maintainer to modify things 2 times (e.g. If I change max credential size from 8 to 9, I would have to update both constant and comment without any obvious gain in readability)

Please fix comments in this fie alltogether. Assume that:
  - file is in efr32/include/LockManager.h . This implies that these are for EFR32 builds, you do not need to tell me this is for EFR32
  - `#define FOO_MAX_BAR n` implies that the maximum BAR value for FOO is n. You do not need to tell me that in a comment.